data_models/environments/dev

```diff
@@ -0,0 +1,8 @@
+DJANGO_SECRET_KEY=!f3p74+hi5omz+)m((+8597&&01h-&bf(0#d1mwvak&clmv*%g
```

Comment:
I agree that the actual risk for a dev server is fairly small. Still, we do much of our development "in the open" and it would be a bummer to find that someone trashed our dev instance database. I see that the same key is also used and exposed for `environments/test`, a server which will almost certainly be exposed to the internet at some point.

A bigger concern is having this key *accidentally* re-used in a more important context. I've tried to prevent that entirely by not having any secrets embedded in source control anywhere. Maybe that's too extreme. What are your thoughts, @dongbohu and @rzelayafavila?